Automattic/pocket-casts-android
# App option to toggle showing podcast artwork embedded in show notes

## Description
Recently, a change was made to show artwork that is embedded in a podcast episode's show notes by default.  This adds a toggle to appearance settings that functions similarly to the existing toggle for embedded artwork.

Fixes #1709

## Testing Instructions
1. Begin playing an episode that includes artwork embedded in show notes (eg. https://pcast.pocketcasts.net/podcast/ede41160-9eeb-012f-3e7d-525400c11844)
2. Observe that the default podcast artwork is shown in both the miniplayer and fullscreen player.
3. Navigate to Settings -> Appearance and toggle "Use episode artwork"
4. Observe that embedded artwork is show in fullscreen and miniplayer (playback state may need to be toggled for miniplayer to be updated)

## Screenshots or Screencast 
<img width="397" alt="Screenshot 2024-02-17 at 11 22 16" src="https://github.com/Automattic/pocket-casts-android/assets/20568035/9299eb13-7655-4138-bdb7-5bbcbb2c8d7d">
<img width="397" alt="Screenshot 2024-02-17 at 11 22 19" src="https://github.com/Automattic/pocket-casts-android/assets/20568035/0b905335-3b29-4f7e-ba5f-0a4185a77b79">
<img width="397" alt="Screenshot 2024-02-17 at 11 22 32" src="https://github.com/Automattic/pocket-casts-android/assets/20568035/78d128cf-f104-49d0-9e3c-1339ed7852f1">
<img width="397" alt="Screenshot 2024-02-17 at 11 22 28" src="https://github.com/Automattic/pocket-casts-android/assets/20568035/f4668fb7-9406-4091-80ee-b93ee856e07d">


## Checklist
 
#### I have tested any UI changes...
